util/secret/consts.go

```diff
@@ -46,4 +46,7 @@ const (
 
 	// APIServerEtcdClient is the secret name of user-supplied secret containing the apiserver-etcd-client key/cert
 	APIServerEtcdClient Purpose = "apiserver-etcd-client"
+
+	// PostApplyDataKey is the key used to store a PostApply addon in the secret's data field.
+	PostApplyDataKey = "addon.yaml"
```

Comment:
Does the name need to be a file? Or could we use something generic like `value` like we do elsewhere?